packages/sources/synthetix-debt-pool/test/integration/__snapshots__/adapter.test.ts.snap

```diff
@@ -3,43 +3,43 @@
 exports[`synthetix-debt-pool debt successfully fetches the current debt size of the synthetix debt cache across "mainnet" and "mainnet-ovm" if chainSources is missing 1`] = `
 Object {
   "data": Object {
-    "result": "313273658056626393719587417",
+    "result": "313273654056626393719587417",
```

Comment:
these values are all going the opposite direction they should be--the debt is higher when there is synths in transit, not lower